modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/TicketAttachmentsTable.jsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import {useModal} from '@clayui/core';
+import {useEffect, useState} from 'react';
+import {PAGE_ROUTER_TYPES} from '~/common/utils/constants';
+import i18n from '../../../../../../../common/I18n';
+import Table from '../../../../../../../common/components/Table';
+import {getTicketAttachments} from '../../../../../../../common/services/liferay/api';
+import useMyUserAccountByAccountExternalReferenceCode from '../../../../Project/TeamMembers/components/TeamMembersTable/hooks/useMyUserAccountByAccountExternalReferenceCode';
+import DeleteTicketAttachmentModal from './components/DeleteTicketAttachmentModal/DeleteTicketAttachmentModal';
+import TicketAttachmentsTableEmpty from './components/TicketAttachmentsTableEmpty';
+import OptionsColumn from './components/columns/OptionsColumn';
+import useDelete from './hooks/useDeleteTicketAttachment';
+import useDownload from './hooks/useDownloadTicketAttachment';
+import usePagination from './hooks/usePaginationTicketAttachments';
+import useSort from './hooks/useSortTicketAttachments';
+import getAttachmentFormattedDateTime from './utils/getAttachmentFormattedDateTime';
+import {getColumns} from './utils/getColumns';
+
+const TicketAttachmentsTable = ({
+	koroneikiAccount,
+	loading: koroneikiAccountLoading,
+}) => {
+	const {
+		data: myUserAccountData,
+		loading: myUserAccountLoading,
+	} = useMyUserAccountByAccountExternalReferenceCode(
+		koroneikiAccountLoading,
+		koroneikiAccount?.accountKey
+	);
+
+	const loading = myUserAccountLoading;
+	const loggedUserAccount = myUserAccountData?.myUserAccount;
+
+	const [ticketAttachments, setTicketAttachments] = useState([]);
+	const [selectedTicketAttachment, setSelectedTicketAttachment] = useState();
+	const [toastItems, setToastItems] = useState([]);
+
+	const {handleSortChange, sortConfig} = useSort();
+	const {
+		paginationConfig,
+		sortedTicketAttachmentsFilteredPerPage,
+	} = usePagination(sortConfig, ticketAttachments);
+	const {onDownload} = useDownload();
+	const {isDeleting, onDelete} = useDelete();
+	const {observer, onOpenChange, open} = useModal();
+
+	useEffect(() => {
+		const fetchTicketAttachments = async () => {
+			const ticketAttachmentsResponse = await getTicketAttachments(
+				koroneikiAccount?.accountKey
+			);
+
+			const ticketAttachmentsData = await ticketAttachmentsResponse.json();
```

Comment:
If you use fetcher parse is not needed

```suggestion
			const ticketAttachmentsResponse = await getTicketAttachments(
				koroneikiAccount?.accountKey
			);
```